components/StateMultipleChoiceQuizPage.tsx

```diff
@@ -1,10 +1,57 @@
-import React from "react";
-import { Text, View } from "react-native";
+import { useTheme } from "@react-navigation/native";
+import React, { useState } from "react";
+import { Text, View, Button } from "react-native";
 
-export default function StateMutipleChoiceQuizPage() {
+type Question = {
+  questionText: string;
+  option: string[];
+  correctAnswer: string;
+};
+const quizQuestions: Question[] = [
+  {
+    questionText: "What is Karnataka's capital?",
+    option: ["Bangalore", "Chennai", "Hyderabad", "Raipur"],
+    correctAnswer: "Bangalore",
+  },
+];
+
+const StateMultipleChoiceQuizApp: React.FC = () => {
+  const [currentQuestionIndex, setCurrentQuestion] = useState<number>(0);
+  const [score, setScore] = useState<number>(0);
+  const [quizFinished, setQuizFinished] = useState<boolean>(false);
+
+  const currentQuestion = quizQuestions[currentQuestionIndex];
+
+  const handleAnswer = (selectedOption: string) => {
+    if (selectedOption === currentQuestion.correctAnswer) {
+      setScore(score + 1);
+    }
+    if (currentQuestionIndex < quizQuestions.length - 1) {
+      setCurrentQuestion(currentQuestionIndex + 1);
+    } else {
+      setQuizFinished(true);
+    }
+  };
   return (
```

Comment:
Add a line break before the `return` statement.

Reply:
I have in the recent commit